### PR TITLE
Tokens Form: Only copy token values that are changed from default

### PR DESCRIPTION
**Pull request by laras126**
In order to manage the values of default tokens in one place, the tokens form should only copy the token values that have changed.

To check this functionality:
1. Navigate to the branch deployment here: https://pmc-larva-git-fix-tokens-diff-penske-media-corp.vercel.app
2. Select "Design Tokens"
2. Create tokens for a new brand with any name.
3. Change the value of any token on the right
4. Click "Save JSON to file". 
5. In the file, ensure there is only one token in the "props" key.

### CODE REVIEW

**Review by sarahsmo — APPROVED** [review]
LGTM!

File: packages/site/src/components/TokensView.js [hunk 1]
```diff
@@ -108,17 +111,53 @@ export const TokensView = () => {
...
 	 * Save JSON to file for browsers that support it, and fallback to
 	 * copy to clipboard for browsers that don't.
 	 *
 	 * @see https://developer.mozilla.org/en-US/docs/Web/API/window/showSaveFilePicker
 	 */
 	const saveJsonToFile = async () => {
+
+		const diffedTokens = ( ( changed, reference ) => {
+			const updatedTokenKeys = Object.keys( changed ).filter( key => {
+				return changed[key].value !== reference[key].value;
```
Comment (ethitter) [inline packages/site/src/components/TokensView.js:146]: This introduces a TypeError when updating an existing brand's tokens. Looks like `reference[key]` is undefined in that case.

**Review by ethitter — CHANGES_REQUESTED** [review]

File: packages/site/src/components/TokensView.js (hunk 1 above)
Comment (laras126) [reply packages/site/src/components/TokensView.js:146]: Good catch! This was happening when there are tokens in the changed object that are not in the reference object - i.e. theme-specific tokens.

**Review by ethitter — APPROVED** [review]
:shipit:

**Pull request merged**